src/main/java/org/ohdsi/webapi/shiro/management/AtlasSecurity.java

```diff
@@ -94,6 +94,8 @@ private void init() {
     if (this.authorizationMode.equals("teamproject")){
       // add system role that enables read restrictions/permissions based read access configurations:
       this.defaultRoles.add("read restricted Atlas Users"); // aka reserved system role 15
+      // temporary solution to simplify onboarding: also add the "Source user (omop)" role to the list of defaultRoles for each user:
+      this.defaultRoles.add("Source user (omop)"); // TODO - replace with final solution from https://ctds-planx.atlassian.net/browse/VADC-1086
```

Comment:
How this will work if the user is not in the useryaml?

Reply:
Strictly speaking, this is not related to the user yaml file at all. This is just to avoid the extra step of having to manually add the "source user" role to a user after it is already on the yaml and using the system.

Reply:
our Atlas is running in "teamproject" authorization mode. So, if the user does not pass in a teamproject on which he is granted permission (in Arborist), then he will not be able to use Atlas. See [here](https://github.com/uc-cdis/WebAPI/blob/pre_release_2.15_teamproject_dev/src/main/java/org/ohdsi/webapi/shiro/filters/TeamProjectBasedAuthorizingFilter.java#L58-L62) and [here](https://github.com/uc-cdis/WebAPI/blob/pre_release_2.15_teamproject_dev/src/main/java/org/ohdsi/webapi/shiro/filters/TeamProjectBasedAuthorizingFilter.java#L108-L123).

Reply:
Ok, that makes sense. 👍